### PR TITLE
Questions impacts ui

### PR DESCRIPTION
**Pull request by ChaoticLeah**
Made the questions UI. I also made a script that fills in the question and makes the buttons. I made a little test thing that can be accessed by clicking on Questions-Canvas, and in the inspector, you will see a few things that look like this:
image
You can change the fields, the amount of them, and the question from there. Don't touch these:
image
They control what text it will modify and what button it will clone and edit.
Once you set those things you can run it with the play button and it should all get done correctly.
image
(I also did the pixel art, so it was not taken from google)

### CODE REVIEW

File: Assets/Scripts/QuestionButtonAmt.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            ButtonClone.transform.position = position;
+            ButtonText = ButtonClone.GetComponentInChildren(typeof(Text)) as Text;
+            ButtonText.text = answers[i];
+            ButtonClone.transform.parent = GameObject.Find("Questions-Canvas/Choice 1").transform.parent;
+        }
+
+    }
+
+    // Update is called once per frame
+    void Update()
```
Comment (kronodeus) [inline Assets/Scripts/QuestionButtonAmt.cs]: It is OK to remove the `Update` function since we are not using it in this script

Comment (ChaoticLeah) [reply Assets/Scripts/QuestionButtonAmt.cs]: Oh, I didn't know that, how should I do that since I already made the request?

Comment (ChaoticLeah) [reply Assets/Scripts/QuestionButtonAmt.cs]: Ok I did that

**Review by eliyahubowers — APPROVED** [review]
I approve of this. You can make it more proportional by changing the rect transform on the panel to stretch setting left top bottom right to 0 and then changing the min and max anchors.

**Review by kronodeus — APPROVED** [review]

**Pull request merged**